samples/Server/appsettings.json

```diff
@@ -8,5 +8,6 @@
   },
   "AllowedHosts": "*",
   "ResourceConnectionString": "REPLACE_WITH_CONNECTION_STRING",
-  "EndpointUrl":"REPLACE_WITH_ENDPOINT_URL"
+  "EndpointUrl":"REPLACE_WITH_ENDPOINT_URL",
+  "AdminUserId": "REPLACE_WITH_ADMIN_USER_ID"
```

Comment:
update readme instructions on populating this new setting

Reply:
Also add details to the readme on what an admin user id is for in this sample application